src/command_modules/azure-cli-monitor/azure/cli/command_modules/monitor/params.py

```diff
@@ -153,3 +153,29 @@ def register_resource_parameter(command, dest, arg_group=None, required=True):
     c.argument('end_time', arg_group=filter_arg_group_name)
     c.argument('caller', arg_group=filter_arg_group_name)
     c.argument('status', arg_group=filter_arg_group_name)
+
+with ParametersContext(command='monitor action-group show') as c:
+    c.register_alias('action_group_name', ('--name', '-n'))
+
+with ParametersContext(command='monitor action-group delete') as c:
+    c.register_alias('action_group_name', ('--name', '-n'))
+
+with ParametersContext(command='monitor action-group create') as c:
+    from .validators import process_action_group_detail_for_creation
+    from .actions import ActionGroupReceiverParameterAction
+
+    c.register_alias('action_group_name', ('--name', '-n'), validator=process_action_group_detail_for_creation)
+    c.extra('receivers', options_list=('--add-receiver', '-a'), nargs='+', arg_group='Receivers',
+            action=ActionGroupReceiverParameterAction)
+    c.extra('short_name')
+    c.extra('tags')
+    c.ignore('action_group')
+
+with ParametersContext(command='monitor action-group update') as c:
+    c.register_alias('action_group_name', ('--name', '-n'))
+    c.ignore('action_group')
+    c.extra('add_receivers', options_list=('--add-receiver', '-a'), nargs='+', arg_group='Receivers',
+            action=ActionGroupReceiverParameterAction)
+    c.extra('remove_receivers', options_list=('--remove-receiver', '-r'), nargs='+', arg_group='Receivers')
+    c.extra('short_name')
+    c.extra('tags')
```

Comment:
If your update flags are from a custom command (which presumably they must be) then you can simply add these to your custom command signature instead of having all of these .extra parameters.

Reply:
will do.